src/pretix/control/forms/organizer.py

```diff
@@ -535,11 +539,19 @@ class CustomerUpdateForm(forms.ModelForm):
 
     class Meta:
         model = Customer
-        fields = ['is_active', 'name_parts', 'email', 'is_verified', 'locale']
+        fields = ['is_active', 'name_parts', 'email', 'is_verified', 'phone', 'locale']
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        if not self.instance.phone and (self.instance.organizer.settings.region or self.instance.locale):
+            country_code = self.instance.organizer.settings.region or get_country_by_locale(self.instance.locale)
+            self.initial['phone'] = "+{}.".format(get_phone_prefix(country_code))
```

Comment:
Might not be a new problem, but this will lead to ``"+None."`` in the case where we know nothing. Does that cause issues?

Reply:
Good catch, will probably throw an NumberParse-Error. So I will better handle this …